cfme/tests/configure/test_access_control.py

```diff
@@ -1464,6 +1506,83 @@ def test_superadmin_tenant_admin_crud(appliance):
     assert not user.exists
 
 
+@pytest.mark.tier(2)
+@test_requirements.multi_tenancy
+def test_tenantadmin_user_crud(new_tenant_admin, request, appliance):
```

Comment:
It looks to me like these two tests can easily be parametrized in a single test case, since the automation steps are very similar. What do you think?

Reply:
I'll be automating more CRUD tests in the future, so I'll parameterize the tests then.